models/signed_latency.go

```diff
@@ -0,0 +1,140 @@
+package models
+
+import (
+	"crypto/ecdsa"
+	"sort"
+	"time"
+
+	"github.com/aukilabs/go-tooling/pkg/errors"
+	"github.com/aukilabs/hagall-common/messages/hagallpb"
+	hwebsocket "github.com/aukilabs/hagall-common/websocket"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type SignedLatency struct {
+	RequestID     uint32
```

Comment:
wonder why these fields need to be public

Reply:
We are using the protobuf encoding for the hash. It simplified the payload